src/platform/plugins/shared/navigation/common/constants.ts

```diff
@@ -14,4 +14,5 @@ export const DEFAULT_ROUTES = {
   es: '/app/elasticsearch/overview',
   oblt: '/app/observabilityOnboarding',
   security: '/app/security/get_started',
+  chat: '/app/workchat',
```

Comment:
Note for future me: this is one of the places we will need to update the path when we add an onboard view